src/chttpd/src/chttpd_node.erl

```diff
@@ -74,7 +86,9 @@ handle_node_req(#httpd{method = 'GET', path_parts = [_, _Node, <<"_versions">>]}
             collator_version => couch_util:version_to_binary(ColVer)
         },
         javascript_engine => JsEngine
-    });
+    },
+    Response = maps:merge(BaseResponse, SearchResponse),
```

Comment:
Having a “search” section kinda implies we could have a different name there besides “clouseau”. But if, say, we enable nouveau, it’s still going to report clouseau as the search component if it can ping it.

Since it’s really only about clouseau might be simpler to just call the section “clouseau”? Or maybe this way it’s more consistent with how other entries look …?

Reply:
I mentioned to Bob that nouveau didn't have an equivalent `nouveau_rpc:version/0`, and asked if it should so it could also be included in this endpoint if present, to which he replied:

> nouveau doesn't have that function atm and I'm not sure it needs one. One goal of nouveau is that the Java side can run elsewhere, not be co-located, so having a couchdb node reporting the version of a remote service feels off to me.

You said:
> if, say, we enable nouveau, it’s still going to report clouseau as the search component if it can ping it.

With the current implementation, it will not report anything about nouveau since it doesn't support the `clouseau_rpc:version/0` endpoint.

> Since it’s really only about clouseau might be simpler to just call the section “clouseau”? Or maybe this way it’s more consistent with how other entries look …?

I thought about it, but yeah I felt it was more consistent that way. Also, if we change our mind about exposing nouveau's version here, there is less to change.